README.md

```diff
@@ -8,6 +8,8 @@ VSCode extension (plugin) that supports the Ion data format.
 ## Functionality 
 
 * Syntax highlighting. 
+* Error reporting. 
+* Formatting. 
```

Comment:
We should add a sentence to each item that this happens on the whole file and not while typing for now.